### ISSUE TITLE
Option to disable cache with makeUseAxios

### ISSUE THREAD

**Issue by fabn** [simoneb/axios-hooks#199]
Following my request in https://github.com/simoneb/axios-hooks/pull/37#issuecomment-607783909 and #34 it would be nice to have this

For anyone interested here's a workaround for current version

```js
# Use a cache with expire time of 50ms
export const useApi = makeUseAxios({axios: api, cache: new LRU({ max: 10, maxAge: 50 })});
```

Thanks

**Comment by simoneb** [comment]
@fabn can you have a look at this PR? https://github.com/simoneb/axios-hooks/pull/204

**Comment by fabn** [comment]
I think is fine, as soon as you release it I'll test it.

Thanks.

**Comment by simoneb** [comment]
@fabn I published a prerelease, axios-hooks@1.10.0-0. Can you try it out before I do an official release?

**Comment by fabn** [comment]
@simoneb I tried it and it's working as expected. Thanks.

**Comment by simoneb** [comment]
@fabn thanks for checking. Now released as 1.10.0
